### DISCUSSION TITLE
Idle Notifier - BUG ? -

### DISCUSSION THREAD

**Discussion by YoungNubby** [runelite/runelite#17541] [Ideas]
Hey there, just started using plugins and I am loving it. Runescape feels like a totally new game. There are some occasions where the hover info of the plugin doesn't describe the tool well enough.

One such instance is the "Idle Notifier". It took some toying with to figure out what each option is for,

However, my real complaint is with box traps. They don't seem to fall under a category. You can get away with movement notifications. I am passively fletching at the same time. It is labeled as an animation action. I haven't tried fishing yet, but I would bet it is categorized. Laying box traps don't currently seem to count as an action like fletching does or fishing might. This look like an oversight to me.

I hope this update gets made in the future. If anyone has a solution I'd love to hear it.

**Comment by YoungNubby** [comment]
Then again, I am never here, so feel free to message me in game. Character name Dante Dali.

**Comment by DominickCobb-rs** [comment]
Laying a box trap isn't an action that players do for extended periods of time. Are you suggesting you need to be notified that you're idle after laying a box trap?

**Comment by YoungNubby** [comment]
That's right.
